src/main/java/mate/academy/bookstore/dto/BookRequestDto.java

```diff
@@ -1,13 +1,23 @@
 package mate.academy.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class BookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
     private String author;
+    @NotBlank
+    @ISBN
     private String isbn;
+    @Min(value = 0)
+    @NotNull

```

Comment:
We can add default message like @NotBlank(message = "FIELD_NAME cannot be empty") just in case if something goes wrong with exceptions handlers